src/main/java/com/gl/donate_receive/donate_receive/dto/ReportCreatedDto.java

```diff
@@ -0,0 +1,11 @@
+package com.gl.donate_receive.donate_receive.dto;
+
+import java.util.UUID;
+
+public class ReportCreatedDto {
+	private UUID itemId;
+	private UUID receiverId;
+	private String comment;
+	private byte[] mediaType;
```

Comment:
It should be something like mediaFile imho

Reply:
As a base for using pictures, I assume to use MultipartFile. Because PostgreSQL doesn't have a mediaType data structure.
As an example for those purposes I use the article:
https://frontbackend.com/spring-boot/spring-boot-upload-file-to-postgresql-database

Reply:
I am talking about the name.